### PR TITLE
 repair the destroyed journal soft link through ceph-disk acti…

### PR DESCRIPTION
**Pull request by houlinfei**
…vate-all or activate command.

Signed-off-by: houlinfei <hou.linfei@h3c.com>

### CODE REVIEW

**Comment by athanatos** [comment]
@dachary Can you take a look at this one?

**Comment by dachary** [comment]
test this please

File: src/ceph-disk/ceph_disk/main.py [hunk 1]
```diff
@@ -3178,6 +3224,12 @@ def activate(
         raise Error('No OSD uuid assigned.')
     LOG.debug('OSD uuid is %s', fsid)
 
+    juuid = read_one_line(path, 'journal_uuid')
```
Comment (dachary) [inline src/ceph-disk/ceph_disk/main.py]: It's legitimate to not have a journal in some cases. If that's the case, it should not fail.

**Comment by dachary** [comment]
@houlinfei ping ?

**Comment by dachary** [comment]
@houlinfei feel free to re-open when you have time to update this pull request

**Pull request closed without merging**